File: class/C1/cpp/reverse.cpp
#include <iostream>
using namespace std;
int main()
{
    int a[1000];
    int i=0;
    while(cin.get()!= '\n')
    {
        cin >> a[i];
        i++;
    }
    for (int j=0, k=i-1; j<i/2; j++,k--)
    {
        int temp = a[j];
        a[j] = a[k];
        a[k] = temp;
    }
    for(int j=0; j<i; j++)  cout << a[j] << " ";
    return 0;
}

File: class/C1/python/reverse.py
def reverse(a):
    d = a.size
    for i in range(0,d/2):
        temp = a[i]
        a[i] = a[d-1-i]
        a[d-1-i] = temp
if __name__ == "__main__":
    a = [1,6,3,7]
    reverse(a)
    for i in a:
        print(a," "),